src/components/dataset/DatasetAccess.jsx

```diff
@@ -30,6 +41,14 @@ function DatasetAccess(props) {
 
   return (
     <Grid container spacing={1}>
+      {showHelp ? (
+        <div className={classes.helpContainer}>
+          <a href="#" className={classes.genericLink}>
```

Comment:
Not sure where this link is supposed to go - Please advise! :)

Reply:
oh, that should make a sidebar pop out.  Check out in the dataset info view:
![image](https://user-images.githubusercontent.com/5633787/192885575-6c0db54f-0fc0-40f0-8746-c72b19894915.png)

Reply:
replying to say that i completely forgot to do this, and am working on adding it now 🙏 sorry!